tutorials/text_processing/README.md

```diff
@@ -1,24 +0,0 @@
-# NeMo Text Processing Tutorials
-
-The NeMo Text Processing module provides support for both Text Normalization (TN) and 
-Inverse Text Normalization (ITN) in order to aid upstream and downstream text processing.
-The included tutorials are intended to help you quickly become familiar with the interface
-of the module, as well as guiding you in creating and deploying your own grammars for individual
-text processing needs.
-
-If you wish to learn more about how to use NeMo's for Text Normalization tasks (e.g. conversion
-of symbolic strings to verbal form - such as `15` -> "fifteen"), please see the [`Text Normalization`](https://colab.research.google.com/github/NVIDIA/NeMo/blob/stable/tutorials/text_processing/Text_Normalization.ipynb)
-tutorial.
-
-If you wish to learn more about Inverse Text Normalization - the inverse task of converting 
-from verbalized strings to symbolic written form, as may be encountered in downstream ASR - 
-consult the [`Inverse Text Normalization`](https://colab.research.google.com/github/NVIDIA/NeMo/blob/stable/tutorials/text_processing/Inverse_Text_Normalization.ipynb) tutorial.
```

Comment:
could you keep the README to have links to the docs and a high-level description of the notebooks? without task description